frontend/android/app/src/main/AndroidManifest.xml

```diff
@@ -42,4 +42,7 @@
     <!--<uses-permission android:name="android.permission.CAMERA" tools:node="remove"/>-->
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" tools:node="remove"/>
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" tools:node="remove"/>
+
```

Comment:
is here a whitespace needed?

Reply:
I think it contributes to a better readability as there is already a space between Location Permission and External Storage Permission